lib/data_update_scripts/20201001173841_add_navigation_links.rb

```diff
@@ -0,0 +1,26 @@
+module DataUpdateScripts
+  class AddNavigationLinks
```

Comment:
This data update script will seed all forems with some base links. The base links are ReadingList, Code of Conduct, Privacy Policy, Terms of Use, and Contact. If the team feels any should be removed or added, drop a note here.